### ISSUE TITLE
dont hardcode tos/privacy/rules/guidelines pages

### ISSUE THREAD

**Issue by PF94** [carrevue/OpenSB#237]

**Comment by nixtyninee** [comment]
change it from being md to part of twig and make it so where it says
squarebracket it uses the site name variable so people can use the default
guidelines and shit for their instances
*Sent from my U.S. Cellular® smartphone*

**Comment by PF94** [comment]
> change it from being md to part of twig

then I have to maintain 4 separate copies of 4 pages (help, community guidelines, tos and privacy policy) which is going to be ass.

plus I already have another idea in mind

<img width="576" alt="Image" src="https://github.com/user-attachments/assets/07e67fd7-90a9-4948-9751-2c431b849169" />

the help page currently says "OpenSBInstanceName" because I intend on having it be dynamically replaced with the instance's name, likely through a twig filter (without converting the entire markdown pages into twig templates). the other pages are squarebracket-specific and they're hardcoded like that cause *normally* on a normal opensb instance it should just show some generic placeholder page but I still haven't bothered with implementing that shit

**Comment by nixtyninee** [comment]
i meant like how the video player is one twig file in the 'default' or
'common' or whatever its called skin folder
*Sent from my U.S. Cellular® smartphone*

**Comment by PF94** [comment]
yeah, because the player doesn't have to be adapted for each frontend. but this is different with markdown pages, because each frontend has different implementations and syntax for context boxes.

**Comment by nixtyninee** [comment]
ah
*Sent from my U.S. Cellular® smartphone*
